pkg/systemd/services/service.jsx

```diff
@@ -21,50 +21,132 @@ import React from "react";
 import { Breadcrumb, BreadcrumbItem } from "@patternfly/react-core/dist/esm/components/Breadcrumb/index.js";
 import { Page, PageSection } from "@patternfly/react-core/dist/esm/components/Page/index.js";
 import { Gallery, GalleryItem } from "@patternfly/react-core/dist/esm/layouts/Gallery/index.js";
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
 
+import { EmptyStatePanel } from "cockpit-components-empty-state.jsx";
 import { ServiceDetails } from "./service-details.jsx";
 import { LogsPanel } from "cockpit-components-logs-panel.jsx";
 import { superuser } from 'superuser';
 import { WithDialogs } from "dialogs.jsx";
 
+import s_bus from "./busnames.js";
+
 import cockpit from "cockpit";
 
 const _ = cockpit.gettext;
 
+function debug() {
+    if (window.debugging == "all" || window.debugging?.includes("service-details"))
+        console.debug.apply(console, arguments);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230308-084716-374aaf58-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)